docs/introduction/media/media.md

```diff
@@ -0,0 +1,16 @@
+# Media
+
+# Blogs
+- [M3: Uber’s Open Source, Large-scale Metrics Platform for Prometheus](https://eng.uber.com/m3/) By Rob Skillington - Aug 7, 2018.
+
+# Recorded Talks
+- [Panel Discussion: Prometheus Long-Term Storage Approaches](https://youtu.be/VvJx0WTiGcA?t=23530) including highlights of M3 stack by Nikunj Agarwal - Aug 9, 2018.
+
+- [Putting billions of timeseries to work at Uber with autonomous monitoring](https://vimeo.com/274821002) By Prateek Rungta - Jun 6, 2018. [Slides](http://bit.ly/m3db-monitorama2018)
+
+# Upcoming Talks
```

Comment:
Do we want to separate "Past" and "Upcoming" sections? Seems like it might be annoying to keep these up to date.

Reply:
meh, fewer sections the better really.

Reply:
leaving as is for now